cmd/cert-checker/main.go

```diff
@@ -188,6 +194,27 @@ func (c *certChecker) checkCert(cert core.Certificate) (problems []string) {
 		problems = append(problems, "Stored digest doesn't match certificate digest")
 	}
 
+	// Run linter
+	linter := new(lintasn1.Linter)
+	errs := linter.CheckStruct(cert.DER)
+	if errs != nil {
+		for _, err := range errs.List() {
+			problems = append(problems, err.Error())
+		}
+	}
+	d, err := certdata.Load(cert.DER)
+	if err != nil {
+		problems = append(problems, err.Error())
+	}
+	errs = checks.Certificate.Check(d)
+	if errs != nil {
+		for _, err := range errs.List() {
+			if err.Error() != "commonName field is deprecated" {
```

Comment:
Do you think its worth trying to upstream a change to make this a warning instead of an error, or to allow disabling just this particular Certificate check by configuration?

Reply:
I think filtering out the error from the list is a reasonable way to configure the fact that we don't really care about it. I don't think certlint has a separation of warnings from errors, though zlint does.

Reply:
Sounds good to me. I agree for this case it seems OK :+1: